setup.py

```diff
@@ -55,8 +55,9 @@
     # installed, specify them here.  If using Python 2.6 or less, then these
     # have to be included in MANIFEST.in as well.
     package_data={
-        'allofplos':['seed.db'],
-        'allofplos.seed_corpus': ['*.xml'],
+        'allofplos': ['starter.db'],
+        'allofplos.starter_corpus': ['*.xml'],
+        'allofplos.tests.testdata': ['*.xml']
```

Comment:
Do you expect users to run the tests? It seems like this is an idea that fits better in a separate PR.

Reply:
basically I'd like this line removed and then i think this is good to be merged.

Reply:
fixed!